lib/config.js

```diff
@@ -153,9 +153,18 @@ function parseDatabaseConfig (prefix) {
   // When using SQLite in-memory database, default to sync enabled
   const sync = castBool(getEnv(prefix, 'DB_SYNC'), uri === 'sqlite://')
 
+  // Knex
+  const knex_env = getEnv(prefix, 'DB_KNEX_ENV') || 'development'
+  // Oracle
+  const connection_user = getEnv(prefix, 'DB_CONNECTION_USER')
+  const connection_password = getEnv(prefix, 'DB_CONNECTION_PASSWORD')
```

Comment:
This line does exactly the same thing as other getEnv() lines in this file, and this code will be exercised by ilp-server integration test.  Therefore, I don't think unit test is necessary.